components/bl-totop-component/README.md

```diff
@@ -0,0 +1,47 @@
+# Totop
+
+Totop is the component that can be used in Backendless [UI-Builder](https://backendless.com/developers/#ui-builder). It allows you to use the up button to scroll up the page or scroll up to a specific page element.
+
+<p align="center">
+  <img src="./thumbnail.png" alt="main thumbnail" width="780"/>
+</p>
+
+## Properties
+
+| Property   | Type                                                | Default value  | Logic                      | Data Binding | UI Setting | Description                                                       |
+|------------|-----------------------------------------------------|----------------|----------------------------|--------------|------------|-------------------------------------------------------------------|
+| Position   | *Select* <br> `'bottom-right'` <br> `'bottom-left'` | `bottom-right` | Type Logic                 | NO           | YES        | Controls the position of the up button on the page.               |
+| Element    | *JSON* <br> `{...}`                                 |                | Element To Scroll To Logic | NO           | NO         | The logic to determine the element to which the scroll will be.   |
```

Comment:
JSON? What we should pass here? 

I think we could leave Type column for this empty, hide it from settings(as you do it) and add an codeless example screenshot where you pass a component in handler